test/child_killing_test.rb

```diff
@@ -62,13 +62,13 @@ def assert_child_not_running(child_pid)
 
   if !defined?(RUBY_ENGINE) || RUBY_ENGINE != "jruby"
     it "old signal handling just kills off the child" do
-      worker_pid, child_pid, result = start_worker(0, false)
+      _worker_pid, child_pid, result = start_worker(0, false)
```

Comment:
what is it that generates warnings about this? Ruby itself doesn't, does it? Is this a Rubocop thing?